packages/babel-plugin-transform-es2015-classes/src/loose.js

```diff
@@ -6,6 +6,25 @@ export default class LooseClassTransformer extends VanillaTransformer {
   constructor() {
     super(...arguments);
     this.isLoose = true;
+    this.aliasInserted = false;
+    this.methodAlias = this.path.scope.generateUidIdentifier("proto").name;
+  }
+
+  _insertProtoAliasOnce() {
+    if (!this.aliasInserted) {
+      const classProto = t.memberExpression(
+        this.classRef,
+        t.identifier("prototype"),
+      );
+      const protoDeclaration = t.variableDeclaration("var", [
+        t.variableDeclarator(t.identifier(this.methodAlias), classProto),
```

Comment:
If you are going to use generateUidIdentifier, then you should just the identifier part and not do the `.name` 😄 

```js
this.methodAlias = this.path.scope.generateUidIdentifier("proto")
 t.variableDeclarator(this.methodAlias, classProto),
```